Firestore/src/support/secure_random_bsd.cc

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Firestore/src/support/secure_random.h"
+
+#include "Firestore/src/support/port.h"
+
+#if HAVE_ARC4RANDOM
+
+#include <stdlib.h>
```

Comment:
nit: unnecessary include?

Reply:
Yeah, but secure_random.h already includes it. (Style guide suggests that relying on includes in the related header file is fine, though it doesn't seem to forbid re-including either.)

Reply:
secure_random.h includes `<stdint.h>` to get a definition for `uint32_t`. This is `<stdlib.h>` which is used to get `arc4random` :-).

Reply:
int; lib; what's the difference? :)

(They're both 3 letters; that aught to be close enough.)